src/screens/Home.tsx

```diff
@@ -1,26 +1,116 @@
 import React from "react";
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, View, Button } from 'react-native';
-import { Text } from '../components/atoms/Text';
+import { StyleSheet, View, Button, Pressable, Text, Image } from 'react-native';
+import { LinearGradient } from "expo-linear-gradient";
+import { useFonts, Inter_500Medium, Inter_600SemiBold, Inter_700Bold } from '@expo-google-fonts/inter';
 
 export default function Home({ navigation }: { navigation: any }) {
+    const [fontsLoaded] = useFonts({
+        Inter_500Medium,
+        Inter_600SemiBold,
+        Inter_700Bold
+      });
+      
+      if (!fontsLoaded) {
+        // You can return a loading screen or a placeholder here until the font is loaded
+        return <View><Text>Loading...</Text></View>;
```

Comment:
This loading should maybe be styled in some way?

As far as I know, this will make "Loading..." appear in the top left corner.